docs/src/main/sphinx/connector/googlesheets.rst

```diff
@@ -133,3 +133,28 @@ SQL support
 The connector provides :ref:`globally available <sql-globally-available>` and
 :ref:`read operation <sql-read-operations>` statements to access data and
 metadata in Google Sheets.
+
+Table functions
+---------------
+
+The connector provides specific :doc:`table functions </functions/table>` to
+access Google Sheets.
+
+.. _google-sheets-query-function:
+
+``query(sheet) -> table``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``query`` function allows you to query a Google Sheet directly without
+specifying it as a named table in the metadata sheet.
+
+For example::
+
+    SELECT
+      *
+    FROM
+      TABLE(
+        sheets.system.query(
```

Comment:
This is not a "query", so we need a better name for the functions. Some possibilities include "read" or "sheet".

Reply:
Sure will switch to "sheet"

Reply:
@martint "id" is not the best name since the parameter is actually an expression.

The expression consists of `${sheet_id}#${sheet_range}`. the default sheet range if one is not provided is `$1:$10000`, but the sheet range can also refer to [named ranges](https://support.google.com/docs/answer/63175?hl=en&co=GENIE.Platform%3DDesktop)

"sheet" is better, but `sheets.system.sheet(sheet => 'blah#blah')` is a mouthful 🤷

Reply:
How about `range` then, since that's what that expression represents?